src/orquestra/sdk/_ray/_ray_logs.py

```diff
@@ -130,6 +130,26 @@ def _get_env_setup_lines(self) -> t.Sequence[str]:
 
         return [line.decode() for line in log_line_bytes]
 
+    def _get_system_log_lines(self) -> t.Sequence[str]:
+        # There is currently no concrete rule for which log files fall into the
+        # category of 'system'. Since the log files exist locally for the user, we
+        # simply point them to the appropriate directory rather than trying to
+        # construct the category for them.
+        system_warning = (
+            "WARNING: we don't parse system logs for the local runtime. "
+            "The log files can be found in the directory "
+            "'~/.orquestra/ray/session_latest/logs."
+        )
+        return [system_warning]
+
+    def _get_other_log_lines(self) -> t.Sequence[str]:
+        other_warning = (
+            "WARNING: we don't parse uncategorized logs for the local runtime. "
+            "The log files can be found in the directory "
+            "'~/.orquestra/ray/session_latest/logs."
```

Comment:
**non-blocking**: WDYT of using the [ray_temp()](https://github.com/zapatacomputing/orquestra-workflow-sdk/blob/b25507680dd01004aef153961d80bebc62392af1/src/orquestra/sdk/_base/_services.py#L34C5-L34C18) helper, as a SSOT? This would prevent us from proving bad advice in scenarios where the RAY_TEMP is in a different location. Can't think of a valid use case today, but it might save our butts in the future.

Reply:
Good idea, done in [0ed1c25](https://github.com/zapatacomputing/orquestra-workflow-sdk/pull/221/commits/0ed1c25c45dc02bc4ca7f21681735cb82f7b5196)